templates/out-of-sync.j2

```diff
@@ -106,9 +107,16 @@ $(document).ready(function () {
                 <span class="text-success" title="added">+{{added}}</span>&nbsp;<span class="text-danger" title="removed">-{{removed}}</span>
             </td>
             <td>{{ d.dependent_depth }}</td>
+            <td>
```

Comment:
```suggestion
            <td data-order="{{len(d.status.sync_prs)}}">
```

Reply:
That will error if there are no PRs. I don't think age (measured by PR number) is interesting, whereas being able to spot duplicates is.